scripts/alert_emails/raw_html.html

```diff
@@ -0,0 +1,1006 @@
+<!DOCTYPE html
+    PUBLIC "-//W3C//DTD XHTML 1.0 Transitional //EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<!--[if IE]><html xmlns="http://www.w3.org/1999/xhtml" class="ie"><![endif]-->
+<!--[if !IE]><!-->
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8=
+<title></title>
+<!--[if !mso]><!--><meta http-equiv=" X-UA-Compatible" content="IE=Dedge">
+        <!--<![endif]-->
+        <meta name="viewport" content="width=device-width">
+        <style type==ext/css">
```

Comment:
Is `==` a thing in html?

Reply:
eek no LOL this is the weird encoding thing popping up